types/evaporate/index.d.ts

```diff
@@ -1,12 +1,117 @@
-// Type definitions for EvaporateJS
+// Type definitions for EvaporateJS 2.1
 // Project: https://github.com/TTLabs/EvaporateJS
-// Definitions by: Andrew Kuklewicz <https://github.com/kookster>, Chris Rhoden <https://github.com/chrisrhoden>
+// Definitions by: Junyoung Clare Jang <https://github.com/ailrun>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
 
 export = Evaporate;
 
+type Diff<T extends keyof any, U extends keyof any> = (
```

Comment:
Might be better to upgrade this package to ts2.8 (since this is a major version change anyway) and use the new `Exclude` type.

Reply:
I fix this.